GitHub.Collectors/Authentication/GitHubAppAuthentication.cs

```diff
@@ -73,13 +68,21 @@ public async Task<string> GetAuthorizationHeaderAsync()
                 }
             }
 
-            string jwt = this.CreateJwt();
-            string installationId = await this.FindInstallationId(jwt).ConfigureAwait(false);
-            string result = await this.ObtainPat(jwt, installationId).ConfigureAwait(false);
-            return result;
+            TokenCredential credential;
+            if (this.useInteractiveLogin)
+            {
+                credential = new InteractiveBrowserCredential();
+            }
+            else
+            {
+                credential = new ManagedIdentityCredential();
+            }
```

Comment:
Why isn't this functionally pushed down to the base? Isn't debugging using interactive browsing something desirable in CodeAsData

Reply:
CodeAsData is running with Data Factory which use ClientCertificateCredential() instead of ManagedIdentityCredential().

Reply:
Made the change, now CodeAsData also has interactive credential.